ModularTegustation/tegu_virology.dm

```diff
@@ -0,0 +1,123 @@
+// Virology "advanced" symptoms.
+/datum/symptom/adv_transmit
```

Comment:
Most diseases seem to be missing symptom_delay_min and symptom_delay_max, while all other diseases (including Eternal Youth) have it

Reply:
What do you mean by this? I am pretty sure diseases with no actions don't need one.

Reply:
I havent messed with disease code much, I'm just going by what the upstream does. Eternal Youth have a min and max delay (though it does technically have an effect as it makes you younger)

Reply:
Yeah, but those "symptoms" don't need it.